docs/gke/configs/env-kubeflow.sh

```diff
@@ -19,5 +19,6 @@ export ZONE=us-east1-d
 # Set config file to the YAML file defining your deployment manager configs.
 export CONFIG_FILE=cluster-kubeflow.yaml
 
-# Get the project number
-export PROJECT_NUMBER=`gcloud projects describe ${PROJECT} --format='value(project_number)'`
+# Set OAuth2 Credentials
+export CLIENT_ID=
```

Comment:
I was actually considering this file as a reference for the variables needed rather than something to be checked into source control. Perhaps, we should put a warning in the documentation regarding the sensitivity of these credentials?